R/utils-json.R

```diff
@@ -264,7 +264,7 @@ getApprovals <- function(data, chain_nm, legend_nm, appr_var_all, month=NULL, po
     appr_dates <- NULL
     chain <- data[[chain_nm]]
     
-    if (!isEmpty(chain$approvals$startTime)) {
+    if (!isEmpty(chain$approvals$startTime) && !is.null(chain$startTime)) {
```

Comment:
I did give that a shot but chain$approvals$startTime can sometimes still exist even if chain$startTime does not, so both have to be checked. I think using isEmptyOrBlank to check both variables is probably the safest method though? I can change that